docker/apache/status.conf

```diff
@@ -1,4 +1,4 @@
 ExtendedStatus on
-<Location /mod_status>
```

Comment:
not sure what ` mod_status` as have not played with Apache web server for a while but will go with your way.

Reply:
That is the path that one would use on the endpoint for the Apache receiver. The OTel receiver and most of the samples defaults to `server-status`, confusingly the name of the module that provides the functionality is actually `mod_status`, see https://httpd.apache.org/docs/2.4/mod/mod_status.html